Formula/qldbshell.rb

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+require_relative '../ConfigProvider/config_provider'
+
+class Qldbshell < Formula
+
+  $config_provider = ConfigProvider.new('qldbshell')
+
+  desc "A CLI for interacting with QLDB ledgers"
+  homepage "https://github.com/awslabs/amazon-qldb-shell"
+  version $config_provider.version
+  bottle :unneeded
```

Comment:
Does this build from source on install?

Reply:
No it downloads the precompiled MacOS binary (it’s a Rust executable).

Reply:
Ok verifying, since it says bottle unneeded.